### DISCUSSION TITLE
Widget parent class api exposition

### DISCUSSION THREAD

**Discussion by valentinMachado** [iTowns/itowns#2062] [Ideas]
[Widget class](https://github.com/iTowns/itowns/blob/master/src/Utils/gui/Widget.js) is not exposed in itowns/widgets api, this does not allow an external project to create inherited widget extending this class (making a possible contribution easier then).

Could we expose it in the API of itowns/widgets ?

**Comment by jailln** [comment]
Yes we should do expose it

**Reply by valentinMachado** [reply]
Ok, I've done a [PR](https://github.com/iTowns/itowns/pull/2068)

**Reply by jailln** [reply]
@valentinMachado can you close this discussion please ?
